### PR TITLE
Bug 1198658 - MMS retrival unable to be rejected because of dataclone error. r=julienw

### PR DESCRIPTION
**Pull request by steveck-chung**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0eae9563fca5913ed09f54ccdbd8c510adca2316

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5d020c6b1b067e597143cbabc2b74ed8c63924ea

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6b8c51383132b53708d84c03f032efd4b2ef5bc5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c5aa3ceecd632c8cfc61b182f358b5c8bb3cabda

File: apps/sms/services/js/moz_mobile_message/moz_mobile_message_shim.js [hunk 1]
```diff
@@ -147,6 +148,10 @@ var MozMobileMessageShim = {
     return mozMobileMessage.retrieveMMS(id).then((message) => {
       // Return void instead of message to avoid clone error issue.
       return;
+    }, (error) => {
+      // Clone the error if error is DOMError. Should be removed once
+      // https://github.com/gaia-components/bridge/issues/80 addressed.
```
Comment (julienw) [inline apps/sms/services/js/moz_mobile_message/moz_mobile_message_shim.js]: actually I think it's easier of we request DOMError (and maybe DOMException) to be cloneable; can you file a bug on Gecko (in Core::DOM) and add the bug number here (but keep the link to the bridge issue) ?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=326d7fdb9966bc8fc17dbc5d88b376e8e3c6df07

**Pull request merged**